ZIGSIMPlus/Models/Commands/TouchMonitoringCommand.swift

```diff
@@ -15,38 +16,57 @@ public final class TouchMonitoringCommand: AutoUpdatedCommand {
     
     public func start(completion: ((String?) -> Void)?) {
         TouchDataStore.shared.callback = { (touches) in
-            var stringMsg = ""
             
-            for touch in touches {
-                var point = touch.location(in: touch.view!)
-                point = Utils.remapToScreenCoord(point)
-
+            guard let isTouchActive = AppSettingModel.shared.isActiveByCommandData[Label.touch],
+                let isApplePencilActive = AppSettingModel.shared.isActiveByCommandData[Label.applePencil]
+                else {
+                    fatalError("AppSetting of the CommandData nil")
+            }
```

Comment:
Do we need to declare `isTouchActive` `isApplePencilActive` here?
It maybe moved to `getTouchResult` to make it simpler.

Reply:
Thanks. I refactored it.